playwright/_impl/_connection.py

```diff
@@ -180,6 +183,15 @@ def call_on_object_with_known_name(
     ) -> None:
         self._waiting_for_object[guid] = callback
 
+    def initialize_playwright(self) -> asyncio.Future["Playwright"]:
```

Comment:
Future is not subscriptable 
```suggestion
    def initialize_playwright(self) -> "asyncio.Future[Playwright]":
```